aws/websites/download.pytorch.org/meta_cdn_whl_redirect.js

```diff
@@ -1,22 +1,66 @@
 function handler(event) {
     var request = event.request;
+    const headers = request.headers;
     var uri = request.uri;
+    var uri_parts = uri.split('/');
+    var last_uri_part = uri_parts[uri_parts.length -1];
```

Comment:
Maybe uri_params?

Reply:
It was inherited from: [main/aws/websites/download.pytorch.org/pep503_whl_redirect.js](https://github.com/pytorch/test-infra/blob/main/aws/websites/download.pytorch.org/pep503_whl_redirect.js?rgh-link-date=2024-05-15T18%3A38%3A58Z) Lets keep it this way for clarity